runtime/fastly/CMakeLists.txt

```diff
@@ -2,7 +2,6 @@ cmake_minimum_required(VERSION 3.27)
 
 include("../StarlingMonkey/cmake/add_as_subproject.cmake")
 
-add_builtin(fastly::runtime SRC handler.cpp host-api/component/fastly_world_adapter.cpp)
```

Comment:
This shouldn't be deleted, as we do need `handler.cpp`, so perhaps just change it to:

```
add_builtin(fastly::runtime SRC host-api/component/fastly_world_adapter.cpp)
```